mapping/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
 
 # Create your models here.
+
+class Country(models.Model):
+    name = models.CharField(max_length=50)
+    # add other fields/metadata here 
+    
+    def __str__(self):
+        return self.name
+
+class TaskDir(models.Model):
+    DEPARTMENT = (
+        ('Health'),
+        ('Organisation Development', 'OD'),
+        ('Disaster Management', 'DM'),
+    )
+
+    country = models.ForeignKey(Country)
+    department = models.CharField(choices=DEPARTMENT)
+    task = models.CharField(max_length=100)
+    subtask = models.CharField(max_length=100)    
+
+    def __str__(self):
+        return self.id
+
+class TaskDesc(models.Model):
+    STATUS = (
+        (1),(2),(3),(4),(5),
```

Comment:
Choices should be a tuple of tuples https://docs.djangoproject.com/en/1.8/ref/models/fields/